### ISSUE TITLE
[QUESTION] Regarding ACS oAuth tokens...

### ISSUE THREAD

**Issue by NetItUp-FrancescoPasqualatto** [KoenZomers/KeePassOneDriveSync#169]
ACS oAuth tokens are going to be retired on 02/04/2026, is this gonna be a problem for this KeePass plugin?

**Comment by KoenZomers** [comment]
Yes it will be. I have just revived the project and spent about a day to modernize the code and get the OneDrive piece to work again. Haven't looked at the lesser used SharePoint part. If there's still interest for it, I can dive into this later.
